Homework2/Program.cs

```diff
@@ -10,6 +10,74 @@ class Program
     {
         static void Main(string[] args)
         {
+            int nominal = 0;
+            double priceOrigin = 0;
+            double priceTrade = 0;
+
+            do
+            {
```

Comment:
Could you please try to refactor this code to decrease nesting? It's usually hard to read and understand methods that have a lot of conditions/loops/try-catch statements nested one into another. 
You can achieve a better result if you'd move some part of the code into the separate methods (e.g. you can move the code inside the main loop `while (Console.ReadKey().Key != ConsoleKey.Escape)` into the separate method, introduce method to ask user to for positive int input via console)